templates/views/intro-video.html

```diff
@@ -0,0 +1,11 @@
+<div class="f-modal-header header">
+    <i class="f-modal-icon icon youtube"></i>
+  VIDEO
+</div>
+<div class="content">
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/GqCc-vgjTfA" frameborder="0" allow="encrypted-media" allowfullscreen></iframe>
```

Comment:
Through some of my own experimentation I found you can generally use style and non-pixel units like `em` for iframes, have you tried that?  I'd like to stay away from hard coded pixel values if possible.   

We change our base font size for different platforms and screen sizes, so being able to use `em` or `vh` / `vw` is generally preferable.